ocs_ci/ocs/resources/pod.py

```diff
@@ -608,12 +615,17 @@ def exec_run_io_cmd(pod_obj, expect_to_fail):
                     return
             raise ex
 
-    thread = Thread(target=exec_run_io_cmd, args=(pod_obj, expect_to_fail,))
+    thread = Thread(target=exec_run_io_cmd, args=(pod_obj, expect_to_fail, fedora_dc))
     thread.start()
     time.sleep(2)
 
     # Checking file existence
-    test_file = TEST_FILE + "1"
+    if fedora_dc:
+        pod_obj.install_packages("findutils")
```

Comment:
how about we move this to check_file_existence() function itself..

Reply:
Done, moved install_packges to  check_file_existence()